doc/rules/alias/modernize_strpos.rst

```diff
@@ -80,6 +80,8 @@ Rule sets
 
 The rule is part of the following rule sets:
 
+- `@PHP8.0Migration:risky <./../../ruleSets/PHP8.0MigrationRisky.rst>`_
+- `@PHP8.2Migration:risky <./../../ruleSets/PHP8.2MigrationRisky.rst>`_
```

Comment:
Not sure if it looks better.  For me it looks like two parts separated by a dot: `PHP8` and `2Migration`.

But of course I agree that unifying the versioned set names is a good idea. And the new style is more flexible (especially for versions like `8.11` etc.)

So I don't really like it, but I also don't have a better idea.